### ISSUE TITLE
[DataTable] add page in datatable

### ISSUE THREAD

**Issue by Brickpsp** [tleunen/react-mdl#336]
I have heavy data need load to table but if i load all, browser will lag, so can we add page in DataTable to limit data load to view? 

Sorry for my bad english.

**Comment by xdvarpunen** [comment]
Reduce the amount of data DataTable renders by giving it X size of array per page to render and when you need to show next X size of array, rerender DataTable element with new given X size of array.

In pseudocode:

```jsx
data = yourBigLoadOfdata.takeFirst10;

<DataTable selectable shadow={0} rowKeyColumn="id" rows={data}>
  <TableHeader name="oneOfYourDataColumns" tooltip="SomeUsefulTip">oneOfYourDataColumns</TableHeader>
</DataTable>
```

**Comment by Brickpsp** [comment]
oh i see,  i will try create my own page navigation and combine your method.

Thank for your help.

**Comment by tleunen** [comment]
Yes currently, the only way is by managing the paging yourself. There's no solution in MDL at this moment afaik. 

Note that the Material Design specs has a section for pagination but we'll have to wait for MDL to add the css for it before implementing it in react-mdl: https://material.google.com/components/data-tables.html#data-tables-specs
